tests/frontend/ui/README.md

```diff
@@ -0,0 +1,200 @@
+# Integration Tests for the [Mozilla Addons Website][amo].
```

Comment:
Is this file still up-to-date?

Reply:
I'll have to go through it, I suspect it is not.

Reply:
If not, then I'd suggest we either remove the file or replace its content with only a few words.